sonar-plugin-api/src/main/java/org/sonar/api/server/ws/WebService.java

```diff
@@ -629,12 +630,26 @@ public NewParam setDeprecatedSince(@Nullable String deprecatedSince) {
 
     /**
      * @since 5.0
+     * @deprecated since 6.4
+     * @see #setDeprecatedKey(String, String) 
      */
+    @Deprecated
     public NewParam setDeprecatedKey(@Nullable String s) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)